packages/block-library/src/image/edit.js

```diff
@@ -107,8 +111,16 @@ export function ImageEdit( {
 		align,
 		metadata,
 	} = attributes;
+
 	const [ temporaryURL, setTemporaryURL ] = useState( attributes.blob );
 
+	const [ resizeListener, sizes ] = useResizeObserver();
```

Comment:
Pretty sure this one is already included in the `Placeholder` component, so it may be best to make these width changes to that component itself, rather than locally here.